ci-operator/README.md

```diff
@@ -362,16 +362,26 @@ where you can determine the namespace where the `ci-operator` is running the job
 
 This will allow to determine the installer logs:
 
-```oc logs -f e2e -c setup  -n ci-op-3167h4v3 ```
+```
+oc logs -f e2e -c setup  -n ci-op-3167h4v3
+```
+
+Note, it is important to supply the name of your github account in order to be
+granted RBAC permission to view pods in the created test namespace.
```

Comment:
And that the job passes `--give-pr-author-access-to-namespace` to `ci-operator` (all this assuming it is a `ci-operator` job).

Reply:
Do I need to update anything about this PR?

Reply:
On second thought, all auto-generated jobs have that flag, so we can leave this out.